acceptance/lib/helper.rb

```diff
@@ -23,18 +23,22 @@ def self.initialize_config(options)
                          nil, "Puppet Server Version",
                          "PUPPETSERVER_VERSION", nil, :string)
 
+    # This value now come from the beaker options file
+    #   the beaker options file can have this value updated via the rake task
     puppet_version = get_option_value(options[:puppet_version],
                          nil, "Puppet Version", "PUPPET_VERSION",
-                         "4.99.0.403.g33cc892",
+                         nil,
                          :string) ||
                          get_puppet_version
 
     # puppet-agent version corresponds to packaged development version located at:
     # http://builds.delivery.puppetlabs.net/puppet-agent/
+    # This value now come from the beaker options file
+    #   the beaker options file can have this value updated via the rake task
```

Comment:
this comment is probably not quite clear enough.  

i believe `#get_option_value` finds a final value from various places.  i changed the default value to nil, as the default should probably come from the options file `options[:puppet_build_version]` below.  but yes, it also provides documentation as well as an env_var override, `PUPPET_BUILD_VERSION` below.